### ISSUE TITLE
[Bug report]: Can't resolve '../images/image.png'

### ISSUE THREAD

**Issue by francoisromain** [vuepress/vuepress-next#33]
### Description

Can not import images from sub directories. 

This makes an error : 

```md
![image](../images/image.png)
```

### Steps to reproduce

- Reproduction link / repo: https://github.com/francoisromain/bug-vuepress-2-image

### Expected behavior

It should be possible to import images from anywhere

### Environment info

- Output of `vuepress info`:

```sh
System:
    Shell: 5.1.4 - /usr/local/bin/bash
  Binaries:
    Node: 15.5.1 - /usr/local/bin/node
    Yarn: Not Found
    npm: 7.3.0 - /usr/local/bin/npm
  Utilities:
    Git: 2.30.0 - /usr/local/bin/git
  Browsers:
    Chrome: 83.0.4103.116
    Edge: Not Found
    Firefox: 84.0.2
    Safari: 13.1.2
  npmPackages:
    @vuepress/bundler-webpack:  2.0.0-alpha.15 
    @vuepress/cli:  2.0.0-alpha.15 
    @vuepress/client:  2.0.0-alpha.15 
    @vuepress/core:  2.0.0-alpha.15 
    @vuepress/markdown:  2.0.0-alpha.15 
    @vuepress/plugin-active-header-links:  2.0.0-alpha.15 
    @vuepress/plugin-back-to-top:  2.0.0-alpha.15 
    @vuepress/plugin-container:  2.0.0-alpha.15 
    @vuepress/plugin-debug:  2.0.0-alpha.15 
    @vuepress/plugin-docsearch: Not Found
    @vuepress/plugin-git:  2.0.0-alpha.15 
    @vuepress/plugin-google-analytics: Not Found
    @vuepress/plugin-medium-zoom:  2.0.0-alpha.15 
    @vuepress/plugin-nprogress:  2.0.0-alpha.15 
    @vuepress/plugin-palette-stylus:  2.0.0-alpha.15 
    @vuepress/plugin-pwa: Not Found
    @vuepress/plugin-pwa-popup: Not Found
    @vuepress/shared:  2.0.0-alpha.15 
    @vuepress/theme-default:  2.0.0-alpha.15 
    @vuepress/theme-vue: Not Found
    @vuepress/utils:  2.0.0-alpha.15 
    vuepress: ^2.0.0-alpha.15 => 2.0.0-alpha.15 
```

**Comment by meteorlxy** [comment]
Thanks for reporting.

**Comment by francoisromain** [comment]
@meteorlxy Thank you for the quick fix. 

After updating to version `2.0.0-alpha.16`, 
in the repro above, 
I can not acces to the `/sub` page from the browser anymore.

**Comment by meteorlxy** [comment]
Are you missing the ending slash? i.e. `/sub/`.

This might be a webpack-dev-server regression in its 4.0.0-beta.0 version

**Comment by meteorlxy** [comment]
Or it's a breaking change of vue-router-next.

I should only break in dev. You'd better open another issue to report it

**Comment by meteorlxy** [comment]
@francoisromain The `/sub` issue has been fixed via cbe4c7f3924c11b751dfefbb01f8fc0528516b3b
